pkg/controllers/applicationset_controller.go

```diff
@@ -67,6 +70,23 @@ func (r *ApplicationSetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	// Log a warning if there are unrecognized generators
+	hasInvalidGenerators, invalidGenerators := invalidateGenerators(&applicationSetInfo)
+	if len(invalidGenerators) > 0 {
+		gnames := make([]string, 0, len(invalidGenerators))
```

Comment:
Nitpick: this is perfectly valid but `gnames := []string{}` is a bit cleaner

Reply:
Done